content/syllabuses/employed-intro-to-tilde-post-skills-test.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Employed intro to Tilde for post skills test
```

Comment:
consited rewording.

Maybe even: 

Employed learnership intro to Tilde: part 2.

The words are just a bit funny